extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/RemoteSyncHandler.java

```diff
@@ -199,7 +199,11 @@ private void handlePut(HttpServerRequest event) {
         event.bodyHandler(new Handler<Buffer>() {
             @Override
             public void handle(Buffer buffer) {
-                hotReplacementContext.updateFile(event.path(), buffer.getBytes());
+                try {
+                    hotReplacementContext.updateFile(event.path(), buffer.getBytes());
+                } catch (Exception e) {
+                    log.error("Failed to update file", e);
```

Comment:
Stupid question: will the filename be in the exception for sure? Or do we need to include it in the error message?